apps/react-18-tests-v9/cypress-react-18.config.ts

```diff
@@ -1,30 +1,29 @@
 import * as path from 'path';
 import { baseConfig } from '@fluentui/scripts-cypress';
 
-// Exclude files that are not compatible with React 18 yet
-const excludedSpecs = [
```

Comment:
Since we are now on latest Cypress, it's a good time to uncomment those and make them fail. So we can fix it on follow-up PRs